src/utils/sqs.ts

```diff
@@ -0,0 +1,32 @@
+import { SqsService } from "./types";
+import {
+  SendMessageCommand, SendMessageCommandOutput,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { logger } from "./logger";
+
+export function sqsService(): SqsService {
+  const send = async function (messageBody: string): Promise<any> {
+    const { AWS_REGION } = process.env;
```

Comment:
I think you can move the env variables and client outside the send function as they don't change between calls so there is no need to get them every time.

Reply:
Will this make any difference as the sqsService function is always called from the controller when sending an event?

Reply:
Not necessarily at this stage I think. Just thinking if we want to add more functions to the service would be nice to pull env variables at the top for clarity

Reply:
I tried this and it doesn't work because the sqsService function just returns the send function, it doesn't return an object with state and a send method.